.github/workflows/check.yml

```diff
@@ -6,12 +6,14 @@ on:
       - '**/Cargo.lock'
       - 'src/**/*'
       - 'benches/**/*'
+      - 'tests/**/*'
   pull_request:
       paths:
       - '**/Cargo.toml'
       - '**/Cargo.lock'
       - 'src/**/*'
       - 'benches/**/*'
+      - 'tests/**/*'
```

Comment:
Super nit, `tests/**` would be sufficient, no?

Reply:
hmm, I thought `tests/**` will grab only files inside tests and all the subdirectories (but not the files in them)?

maybe a macos thing

```bash
list=(/var/log/**/*); echo ${list[@]} | tr -s ' ' '\n' | wc -l
61
list=(/var/log/**); echo ${list[@]} | tr -s ' ' '\n' | wc -l
49
```